articles/quickstart/spa/_includes/_calling_api_preamble.md

```diff
@@ -1,3 +1,7 @@
-It's likely that your single page app will need to consume resources from a data API. Since you are adding authentication to your app, presumably you need to limit access to these resources such that they are only accessible by an authenticated user who is authorized to retrieve them. Auth0 provides [API Authorization](https://auth0.com/docs/api-auth) to accomplish this.
+Some single-page apps use resources from data APIs. You may want to restrict access to those resources, so that only authenticated users with sufficient privileges can access them. Auth0 lets you manage access to these resources using [API Authorization](https://auth0.com/docs/api-auth).
```

Comment:
Should the link be in this form? According to the 'contributing' guide, this link should be in the (/api-auth) form.